### PR TITLE
Rspec setup

### PR DESCRIPTION
**Pull request by andru1989**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -45,6 +45,18 @@ group :development, :test do
...
   gem 'binding_of_caller'
   gem 'meta_request'
+
+  gem "rspec-rails"
+  gem "factory_girl_rails"
+end
+
+group :test do
+  gem "faker", "~> 1.4.3"
+  gem "capybara"
```
Comment (gomayonqui) [inline Gemfile:55]: Are you using capybara for integration tests?

Comment (andru1989) [inline Gemfile:55]: Yes I will use it after I finish model specs

**Comment by gomayonqui** [comment]
If you are not going to use mini_test, try to create the new project like

```
rails new name -T
```

For not installing the default testing framework

File: config/application.rb [hunk 2]
```diff
@@ -22,5 +22,17 @@ class Application < Rails::Application
...
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.generators do|g|
+      g.test_framework :rspec,
+        fixtures: true,
+        view_specs: false,
+        helper_specs: false,
+        routing_specs: false,
+        controller_specs: true,
+        request_specs: false
```
Comment (orendon) [inline config/application.rb:33]: we may need to enable request_specs in the future for integration tests with capybara/webkit

File: Gemfile [hunk 3]
```diff
@@ -45,6 +45,18 @@ group :development, :test do
...
+  gem "database_cleaner", "~> 1.3.0"
+  gem "launchy", "~> 2.4.2"
+  gem "selenium-webdriver", "~> 2.43.0"
```
Comment (orendon) [inline Gemfile:58]: in the future, we may swtich to capybara webkit to speed things up

Comment (gomayonqui) [inline Gemfile:58]: i think we should do it know ;)

**Pull request merged**